new-packages/ts-project/src/state.ts

```diff
@@ -348,6 +361,43 @@ function extractEdgeGroup(
               registerActionBlocks(ctx, blocks, edge.data.actions);
               return;
             }
+            case 'internal': {
+              if (seenInternalProp) {
+                // `reenter` was already seen
```

Comment:
Where was this `reenter` seen in this case?

Reply:
that might happen with:
```ts
internal: true,
reenter: true
```

whereas the other "branch" gets hit with:
```ts
reenter: true,
internal: true
```